### ISSUE TITLE
Verbose flag for `help` command

### ISSUE THREAD

**Issue by rpunkfu** [tj/mmake#10]
Might be cool to have, for a quick glance at what the makefile does instead of going through every command :)

```
$ make help -v

  start:
    Start the dev server.

    Note that the API server must
    also be running.

  api:    
    Start the API server.

  deps:       
    Display dependency graph.

  size:
    Display size of dependencies.
```

**Comment by tj** [comment]
SGTM!

**Comment by rpunkfu** [comment]
Closed by #11 :)
